### ISSUE TITLE
No code completion in Phphstorm and VS Code on windows with tailwindCSS v^3.2.0

### ISSUE THREAD

**Issue by JensvandeWiel** [tailwindlabs/tailwindcss#9657]
**What version of Tailwind CSS are you using?**

v3.2.0 and v3.2.1

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss

**What version of Node.js are you using?**

nodejs 19 but i've tested 16.18.0 too

**What browser are you using?**

N/A

**What operating system are you using?**

Windows

**Reproduction URL**
N/A

**Describe your issue**

I added tailwindcss to a laravel project trough the guides and there is no code completion in vs code with plugin and with phpstorm.
Versions 3.1.8 and below work fine

**Comment by arifur-rahman-arif** [comment]
@JensvandeWiel Did you added the VScode tailwind plugin

**Comment by DaneParcJr** [comment]
I'm having the same issue while using:

Webstorm 2022.2.3
Webstorm Tailwind Plugin 222.4345.14
Vite 3.10
PostCSS 8.4.18
AutoPrefixer 10.4.12
TailwindCSS 3.2.1

How it's currently setup:

`postcss.config.cjs`
<img width="440" alt="Screen Shot 2022-10-26 at 10 03 51 AM" src="https://user-images.githubusercontent.com/45432721/198047463-46a993fc-1446-4f0a-ab3c-05d5ef4951a1.png">

`tailwind.config.cjs`
<img width="617" alt="Screen Shot 2022-10-26 at 10 04 52 AM" src="https://user-images.githubusercontent.com/45432721/198047688-79ab7266-6135-461d-98e7-4d0cf4b10df7.png">

And the result in `index.html`
<img width="673" alt="Screen Shot 2022-10-26 at 10 05 47 AM" src="https://user-images.githubusercontent.com/45432721/198047895-a2f2d8a2-6c70-4fc9-9fe3-dd142f655d59.png">

**Comment by JensvandeWiel** [comment]
![image](https://user-images.githubusercontent.com/85284773/198582730-c53ec0c4-3e13-418d-a97c-05797d3f58e1.png)

**Comment by wrench1815** [comment]
Yes. Just got hit by the Issue. This is what TailwindCSS extension output says.
```
[Error - 2:57:07 PM] Tailwind CSS: Cannot read properties of null (reading '__isOptionsFunction')
TypeError: Cannot read properties of null (reading '__isOptionsFunction')
```

**Comment by wrench1815** [comment]
> ![image](https://user-images.githubusercontent.com/85284773/198582730-c53ec0c4-3e13-418d-a97c-05797d3f58e1.png)

can you link to the issue? Thanks

**Comment by ouedyan** [comment]
> > ![image](https://user-images.githubusercontent.com/85284773/198582730-c53ec0c4-3e13-418d-a97c-05797d3f58e1.png)
> 
> can you link to the issue? Thanks

Thinks it is this one https://youtrack.jetbrains.com/issue/WEB-53284/Tailwind-3-Completion-for-classes-doesnt-work#focus=Comments-27-6556869.0-0

**Comment by ouedyan** [comment]
> > > ![image](https://user-images.githubusercontent.com/85284773/198582730-c53ec0c4-3e13-418d-a97c-05797d3f58e1.png)
> > 
> > 
> > can you link to the issue? Thanks
> 
> Think it is this one [youtrack.jetbrains.com/issue/WEB-53284/Tailwind-3-Completion-for-classes-doesnt-work#focus=Comments-27-6556869.0-0](https://youtrack.jetbrains.com/issue/WEB-53284/Tailwind-3-Completion-for-classes-doesnt-work#focus=Comments-27-6556869.0-0)

Precisely: https://youtrack.jetbrains.com/issue/WEB-57745/Tailwind-no-completion-when-using-tailwindcss320-Cannot-read-properties-of-undefined-reading-modifier

**Comment by same31** [comment]
Same issue with Tailwind > 3.1.8, Mac OS X Node v16.17.1

**Comment by jvrviegas** [comment]
Same issue here using Neovim with @tailwindcss/language-server, i've downgraded to 3.1.8 and works again. Waiting for the fix...

**Comment by thecrypticace** [comment]
@jvrviegas what version of Tailwind CSS and the language server are you using?

**Comment by jvrviegas** [comment]
@thecrypticace when i have installed the Tailwindcss v3.2.0, @tailwindcss/language-server v0.0.10 and Node v16.18.0

**Comment by wrench1815** [comment]
Any ETA? When will it be fixed? it's already over 2 weeks now.

**Comment by same31** [comment]
> Any ETA? When will it be fixed? it's already over 2 weeks now.

This is not something TailwindCSS has to fix, it's in JetBrains hands, it will be fixed in 2022.3.3 https://youtrack.jetbrains.com/issue/WEB-57745/Tailwind-no-completion-when-using-tailwindcss320-Cannot-read-properties-of-undefined-reading-modifier

**Comment by adamwathan** [comment]
Hey folks! Going to close this one as it looks like JetBrains has updated their plugin now, and we've also tweaked some code in Tailwind to make it so the old plugin wouldn't crash anymore, so as long as you are on the latest version of everything you should be good to go here now 👍

If anyone still has issues feel free to open a new issue with all of the relevant version details and stuff and we can take a look.
